src/App.jsx

```diff
@@ -14,70 +15,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodSelected, selectGood] = useState('Jam');
```

Comment:
goodSelect it's sound as a state of good. I propose you to rename it for selectedGood 
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```